ansible/group_vars/all/labinfo.json

```diff
@@ -21,7 +21,7 @@
         "Arista": {
             "user": "admin",
             "passwd": ["password", "123456"],
-            "enable": ['', null]
+            "enable": ["", null]
```

Comment:
The labinfo.json is not standard because it uses a single quote which is not allowed by json module. Although ast.literal_eval can by pass this issue, i still think it is make sense to update the json file itself instead of use another module